src/connectors/rtl-plus-musik.ts

```diff
@@ -0,0 +1,20 @@
+export {};
+
+Connector.playerSelector = 'plus-player-container';
+
+Connector.trackSelector = 'plus-player-container .album';
```

Comment:
This seems very odd, so just to be sure this is not a typo, can you confirm that the track title is in fact contained in an element of class album?

Reply:
Confirm, it's not a typo. The track name is in an element of class `album` because it *links to* the album the track is part of.

Reply:
Understood, just wanted to make sure just in case it was a mistake 😊